### PR TITLE
Aligning descriptions with the VSCode extension

### PR DESCRIPTION
**Pull request by cbjuan**
# Description

Adding some minor changes to texts to align them exactly to the VSCode extension

## Type of change

Docs

# How Has This Been Tested?

NA

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by vabarbosa — APPROVED** [review]

File: README-PyPi.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Qiskit Code Assistant (Beta)
 
-> This experimental feature is only available, as of today, for some IBM Quantum users.
-> If you are not part of that cohort of users, you can still install this extension; however you will not be able to use the assistant.
-> The Qiskit Code Assistant is a preview feature release, subject to change.
+> This experimental feature is only available, as of today, to IBM Quantum premium users.
+> If you are not part IBM Quantum premium plan, you can still install this extension; however you will not be able to use the assistant.
```
Comment (ajbozarth) [inline README-PyPi.md:4]: the grammar here feels off now, it should probably be "part **_of an_** IBM Quantum premium plan"

**Review by ajbozarth — COMMENTED** [review]
One grammar nit, and is there a reason you're refreshing the yarn lock on a doc update? Is there a patch update we need to catch on some dependency?

File: README-PyPi.md (hunk 1 above)
Comment (cbjuan) [reply README-PyPi.md]: Fixed both comments

File: README-PyPi.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Qiskit Code Assistant (Beta)
 
-> This experimental feature is only available, as of today, for some IBM Quantum users.
-> If you are not part of that cohort of users, you can still install this extension; however you will not be able to use the assistant.
-> The Qiskit Code Assistant is a preview feature release, subject to change.
+> This experimental feature is only available, as of today, to IBM Quantum premium users.
+> If you are not of the IBM Quantum premium plan, you can still install this extension; however you will not be able to use the assistant.
```
Comment (ajbozarth) [inline README-PyPi.md:4]: you accidentally left out the word "part"
```suggestion
> If you are not part of the IBM Quantum premium plan, you can still install this extension; however you will not be able to use the assistant.
```

Comment (ajbozarth) [reply README-PyPi.md:4]: actually the more I read the worse this sounds, we should reword it entirely
```suggestion
> If you do not have an IBM Quantum premium plan, you can still install this extension; however you will not be able to use the assistant.
```

**Review by ajbozarth — APPROVED** [review]
This LGTM, do we need to cut a release for this or can it just go out the next time we release (I would assume in the next couple weeks when jupyterlab 4.3 releases)? If we need a release if you could increment to 0.3.2

**Comment by cbjuan** [comment]
Increased version to 0.3.2

**Pull request merged**